io_scene_godot/converters/animation.py

```diff
@@ -160,8 +160,15 @@ def add_default_animation_resource(self, escn_file, action):
     def create_animation_resource(self, escn_file, action):
         """Create a new animation resource and add it into escn file"""
         new_anim_resource = AnimationResource()
-        resource_id = escn_file.add_internal_resource(
-            new_anim_resource, action)
+
+        # add animation resource without checking hash,
+        # blender action is in world space, while godot animation
+        # is in local space (parent space),  so identical actions
+        # are not necessarily generates identical godot animations
+        escn_file.internal_resources.append(new_anim_resource)
```

Comment:
Hmm. I'm not too happy with this. It's dodging the intended API. I would prefer if you could find something sane to hash. If not, a better solution would be to create a `force_create_internal_resource()` function

Reply:
I didn't create a new API like `force_create_internal_resource()` because I think add a resource without hash is the last thing to do when there is no related hashable object can be found. 

Yeah, you have a point in considering future support for other forms of resources, I will fix it

Reply:
I haven't change `internal_resources` to private, it needs to be public to provide access to resource instances (I already need this in material PR), and I think it would be a little verbose to create a API to access resource instances